src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
`box-sizing: border-box;` is duplicated, so make sense to move it the `*{ box-sizing: border-box; }`